siteResources.js

```diff
@@ -0,0 +1,25 @@
+export const urls = [
+    {
+        name: 'Booklog',
+        url: (isbn) => `https://booklog.jp/item/1/${isbn}`,
+        waitSelector: '.item-info-author',
+        fields: {
+            title: '[itemprop=name]',
+            author: '[itemprop=author]',
+            publisher: '[itemprop=publisher]',
+            about: '[itemprop=description]',
+            pageCount: '.info-area > ul > li',
+        },
+    },
+    {
+        name: 'Bookfinder',
+        url: (isbn) => `https://www.bookfinder.com/search/?isbn=${isbn}&mode=isbn&st=sr&ac=qr`,
```

Comment:
Instead of creating the URLs conditionally in the script, it seemed easier to just move the logic of concatenation to the config objects for the sites.